.circleci/config.yml

```diff
@@ -7,6 +7,10 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: Initialize submodules
+          command: git submodule init && git submodule update --remote
```

Comment:
I believe you can use just one command:

```
git submodule update --init
```

Or

```
git submodule update --init --recursive
```

Reply:
In fact, the submodule approach does not actually work (with either my or your commands). I've just replaced with a clone.